src/bepasty/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "1.2.2"
```

Comment:
don't do that, bepasty uses `setuptools_scm` to determine the version number.

you either need to start from a full git checkout with all tags for this to work, or install a bepasty release from pypi.